ui/src/saros/stf/server/bot/jquery/ISelector.java

```diff
@@ -0,0 +1,114 @@
+package saros.stf.server.bot.jquery;
+
+import org.apache.commons.lang3.StringUtils;
+
+public interface ISelector {
```

Comment:
My guess would be that this was copied from the swt-browser-improved. If this is the case, this should be mentioned in the class javadoc with a reference to the repo in order to provide context to other people looking at the class. If this is true for the whole package, you could also add a short note as package-info (and to the other class).

Subsequently, you could remove the `@author` tags from the class as they are not generally used in this project.

Reply:
I will add the information to `ISelector` `AbstractJavaScriptFunction` and `NoCheckedExceptionCallable` and remove `@author`